### PR TITLE
[flink] Specify the MongoDB CDC version

### PR DESCRIPTION
**Pull request by MonsterChenzhuo**
### Purpose

specify the MongoDB CDC version

### CODE REVIEW

File: docs/content/how-to/cdc-ingestion.md [hunk 1]
```diff
@@ -450,7 +450,7 @@ Synchronization from multiple Kafka topics to Paimon database.
 ### Prepare MongoDB Bundled Jar
 
 ```
-flink-sql-connector-mongodb-*.jar
+flink-sql-connector-mongodb-cdc-2.4.1.jar
```
Comment (JingsongLi) [inline docs/content/how-to/cdc-ingestion.md:453]: Document only cdc 2.4+ is supported.

Comment (MonsterChenzhuo) [reply docs/content/how-to/cdc-ingestion.md:453]: > Document only cdc 2.4+ is supported.

Should it be changed to `flink-sql-connector-mongodb-cdc-2.4.*.jar` like this?

Comment (JingsongLi) [reply docs/content/how-to/cdc-ingestion.md:453]: It is ok to keep this as it is.
I mean you can add another word to explain "only cdc 2.4+ is supported"

Comment (MonsterChenzhuo) [reply docs/content/how-to/cdc-ingestion.md]: done.

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**